pkg/cli/image/append/append.go

```diff
@@ -220,13 +231,10 @@ func (o *AppendImageOptions) Run() error {
 		return err
 	}
 	fromOptions := &imagesource.Options{
-		FileDir:         o.FileDir,
+		FileDir:         o.FromFileDir,
```

Comment:
I’m confused why the PR says the flags weren’t wired when they appear here?

the convention of all image commands is —dir is the default for sources or destinations of type file://.  —from-dir overrides source only and —to-dir usually overrides target (unless the command already had a flag).

Removing consistent behavior for —dir would not be desirable.  Can you describe what you mean by “not wired”?

Reply:
with `oc image extract` the `--dir` flag does not do anything, afaict. It's an imagesource.Options.FileDir but that isn't used anywhere, since when extract, the ref.Type is Registry not File

Reply:
the PR notes --dir is not wired properly for extract

Reply:
`$ oc image extract docker.io/library/busybox:latest --dir busybox` will a) fail to run unless you are in an empty directory or pass the --confirm and b) the dir value isn't used

Reply:
and with append:
`$ oc image append --from file://busybox:latest --from-dir $(pwd)/busybox --to file://busybox-new:hello` 
this works properly, but should it be possible to allow for designating the directory for the appended image? as/is, the only choice it for $(pwd)/v2 . you're right, append isn't broken, but imo flags could be more clear

Reply:
I've removed the changes to append, since nothing is broken there, but I'm adding a few examples. I'll update the referenced bugzilla to clarify

Reply:
I've removed changes to info, also, since it's intended that --dir can either mean read from or copied to.

Reply:
OH! I finally see how the --dir flag does work with extract, it's like this (from master branch of oc):
```console
$ oc image mirror docker.io/library/busybox:latest file://busybox:local --dir busybox-local
(this results in a busybox-local/v2/busybox... contents)
$ mkdir ext; cd ext; oc image extract file://busybox:local --dir ../busybox-local
(this results in extracting image contents to current directory from the busybox-local directory where the mirrored image is stored)
```
If this is a reasonable use-case we should definitely keep this. Should the `--to-dir` option be added though, since every time I use `oc image extract` I'm annoyed that I have to make a clear directory first or run w/ `--confirm` and if I accidentally run w/ `--confirm` by arrowing up I get a slew of unwanted files extracted to my current directory that I then have to clean up.